src/store/stateSelectors.js

```diff
@@ -35,6 +35,10 @@ export function selectWalletsAddresses(state: AppState): WalletsAddressesState {
   return state.walletsAddresses
 }
 
+export function selectWalletsAddressesList(state: AppState): string[] {
+  return state.walletsAddresses.addresses
```

Comment:
Property `state.walletsAddresses` always exists?

Reply:
I don't know, really. We have a very level of entropy in selectors.